### PR TITLE
Incorporate edits on APIC-EM basic lab files

### PR DESCRIPTION
**Pull request by charleslavoy**

### CODE REVIEW

File: labs/apic-em-basic/3.md [hunk 1]
```diff
@@ -1,13 +1,14 @@
-# Lab 2: Network Device Related APIs
+# Lab 2: Use network device-related APIs
 
-## Objectives
+## Objective
```
[the remark this answers is not in the document]
Comment (charleslavoy) [reply labs/apic-em-basic/3.md]: Done.

File: labs/apic-em-basic/1.md [hunk 2]
```diff
@@ -32,17 +27,17 @@ Download and install Git from [https://git-scm.com/downloads]
...
+For macOS, use these commands:
 ```bash
 sudo curl -O https://raw.githubusercontent.com/pypa/pip/master/contrib/get-pip.py
 sudo python3 get-pip.py
 pip3 install requests
 ```
 
 For Windows, complete these steps:
-1. Pip comes with Python 3.4 on Windows. Check to see if Pip is in your python 3 path by entering:
+1. pip comes with Python 3.4 on Windows. Check to see if pip is in your python 3 path by entering:
```
[the remark this answers is not in the document]
Comment (charleslavoy) [reply labs/apic-em-basic/1.md]: Done.

File: labs/apic-em-basic/3.md [hunk 3]
```diff
@@ -104,7 +105,7 @@ print (tabulate(device_list, headers=['number','hostname','ip','type'],tablefmt=
 
 ![Figure](/posts/files/apic-em-basic/network_device.png)
 
-### Output: Device Information in Tabular Format
+#### Output: Device Information in Tabular Format
```
[the remark this answers is not in the document]
Comment (charleslavoy) [reply labs/apic-em-basic/3.md]: Done.

File: labs/apic-em-basic/3.md [hunk 4]
```diff
@@ -532,22 +533,22 @@ This output is the result of the following interactions between the application
...
-`GET /network-device/{id}` - Gets the network device for the given device ID <br>
-`GET /license-info/network-device/{deviceId}` - Gets the list of licenses for network device with given device ID. <br>
-`DELETE /network-device/{id}` - Removes a specific network device from inventory<br>
+*  `GET /network-device/{id}/location` - Gets the location for the given device ID.
+*  `GET /network-device/{id}` - Gets the network device for the given device ID.
+*  `GET /license-info/network-device/{deviceId}` - Gets the list of licenses for network device with given device ID.
+*  `DELETE /network-device/{id}` - Removes a specific network device from inventory.
 
-### <center>This concludes Part I of APIC-EM REST API Learning Labs</center>
+### This concludes Part I of APIC-EM REST API Learning Labs
```
[the remark this answers is not in the document]
Comment (charleslavoy) [reply labs/apic-em-basic/3.md]: Good points. I wasn't sure if there is a boilerplate sentence that is used at the conclusion of the labs.

**Comment by paulzim** [comment]
Looks like you got Anne's corrections. Thanks!

**Pull request merged**